lrauv_ignition_plugins/hooks/hook.dsv.in

```diff
@@ -1,4 +1,3 @@
-prepend-non-duplicate;IGN_LAUNCH_CONFIG_PATH;@CMAKE_INSTALL_PREFIX@/share/@PROJECT_NAME@/launch
 prepend-non-duplicate;IGN_LAUNCH_PLUGIN_PATH;@CMAKE_INSTALL_PREFIX@/lib
```

Comment:
I think this one can be removed too, we're not installing any `ign-launch` plugins